src/middleware/permissionsMiddleware.js

```diff
@@ -1,17 +1,8 @@
-import { rule, shield, and, or, not, allow } from 'graphql-shield'
+import { rule, shield, allow } from 'graphql-shield'
 
-const isAuthenticated = rule()(async (parent, args, ctx, info) => {
-  return ctx.user !== null
```

Comment:
Do not delete those checks!!!

Reply:
They are defined but never used.

Reply:
Sure but in this case, please just comment them out as we will need them very soon and will need the reference! Thanks.

Reply:
Sorry for being so stubborn :laughing: but again I wouldn't. Unused code will confuse other developers and make them think it is required. I would even apply that to code comments to remove clutter. In this particular case, the code we're talking about is broken because of upcase/lowercase inconsistency here and in the seeds.

If you have to:
`git checkout 15d9178369bd29df40f3e2810f8e8eb184c7a5e0^ src/middleware/permissionsMiddleware.js`

Reply:
I you place a todo in front of it, no one will be confused. I think devs can handle comments 😂